packages/optimizer/lib/transformers/PreloadHeroImage.js

```diff
@@ -320,6 +316,7 @@ class PreloadHeroImage {
       }
     }
     node.attribs['i-amphtml-ssr'] = '';
+    node.attribs['data-hero'] = '';
```

Comment:
Is this added just so developers are aware of the element that was identified as the hero? It's not required for validation as far as I can tell.

Reply:
Correct!